soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -60,6 +60,10 @@ config I2C_ITE_IT8XXX2
 	default y
 	depends on I2C
 
+config I2C_ITE_ENHANCE
+	default y
+	depends on I2C
```

Comment:
please default based on dt_compat_status so that this can be removed.

Reply:
Removed.